src/components/Home/ActivityTickerFilter.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react'
+
+import { QueryObserverResult, RefetchOptions, RefetchQueryFilters } from '@tanstack/react-query'
+import classNames from 'classnames'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+
+import useFormatMessage from '../../hooks/useFormatMessage'
+import { ActivityTickerEvent } from '../../shared/types/events'
+import Counter from '../Common/Counter'
+import Sort from '../Icon/Sort'
+
+import './ActivityTickerFilter.css'
+import ActivityTickerFilterItem from './ActivityTickerFilterItem'
+
+export type TickerFilter = {
+  proposals_created: boolean
+  votes: boolean
+  delegation: boolean
+  comments: boolean
+  project_updates: boolean
+}
+
+export const INITIAL_TICKER_FILTER_STATE: TickerFilter = {
+  proposals_created: false,
+  votes: false,
+  delegation: false,
+  comments: false,
+  project_updates: false,
+}
+
+function countTrueProperties(obj: Record<string, boolean>): number {
+  let count = 0
+  for (const key in obj) {
+    if (Object.hasOwn(obj, key)) {
+      if (obj[key]) {
+        count++
+      }
+    }
+  }
+  return count
+}
+
+interface Props {
+  setFilterState: React.Dispatch<React.SetStateAction<TickerFilter>>
+  filterState: TickerFilter
+  refetch: (
+    options?: (RefetchOptions & RefetchQueryFilters) | undefined
+  ) => Promise<QueryObserverResult<ActivityTickerEvent[]>>
+}
+
+export default function ActivityTickerFilter({ setFilterState, filterState, refetch }: Props) {
+  const t = useFormatMessage()
+  const [isOpen, setIsOpen] = useState(false)
+  const selectedFiltersCount = countTrueProperties(filterState)
+
+  const onApply = () => {
+    refetch().then(() => setIsOpen(false))
```

Comment:
If we have a loader in the activity ticker then I wouldn't for the refetch to finish.